environments/common/ecr/locals.tf

```diff
@@ -0,0 +1,8 @@
+locals {
+  applications = [
+    "frontend",
+    "backend",
+    "duty-calculator",
+    "admin"
```

Comment:
You okay to add the search-query-parser too?

Reply:
Can't believe I forgot that. Of course.